src/pages/CreateEventPage/components/isFormFilled.ts

```diff
@@ -0,0 +1,9 @@
+function isFormFilled (name: string, description: string, eventType: string) {
+    if (name !== "" && description !== "" && eventType !== "") {
+      return true
+    } else {
+      return false
+    }
+  }
+
+  export default isFormFilled
```

Comment:
Why is this one in its own file while the `submitEvent` function is not 😅. For consistency reasons, either define both of them in the same file as the component or create a `helpers.ts` file where both of them may live. Personally, I would go for the first option though, since these helper functions aren't really that big.